### PR TITLE
TheShowRoom

### PR DESCRIPTION
**Pull request by Antontall**
## Netlify link
https://65dce4cac231e91b71df7a17--zesty-nasturtium-9850de.netlify.app/

## Collaborators
Did this one solo.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,406 +1,205 @@
...
+      <p class="card-info">${show.genre}</p>
+      <p class="card-info">${show.rating}</p>
+      <p class="description">${show.description}</p>
+    </div>
+  `;
+  });
+};
+
+//Load cards when page opens
+loadShows(shows);
```
Comment (Maria-Manuela) [inline script.js:105]: Strengths:

Clear function name: `loadShows` accurately reflects the function's purpose.
Efficient array iteration: Using `forEach` effectively iterates through the shows array.
Concise string interpolation: The use of template literals () for string interpolation makes the code cleaner and easier to read.

To be considered: 

Missing alternative text for images: Adding alt attributes to the  elements with descriptive text would improve accessibility for users who rely on screen readers.

File: script.js [hunk 2]
```diff
@@ -1,406 +1,205 @@
...
+
+//Declare lowToHigh variable for sort functin to work
+let lowToHigh = true;
+
+//Make buttons return to default styling after clicking
+document.querySelectorAll('.buttons').forEach(button => {
+  button.addEventListener('click', () => {
+    button.blur();
```
Comment (Maria-Manuela) [inline script.js:113]: `button.blur()` interesting method, what does it do? 🤔

File: script.js [hunk 3]
```diff
@@ -1,406 +1,205 @@
...
+    return ascending ? titleA.localeCompare(titleB) : titleB.localeCompare(titleA);
+  });
+};
+
+//Function and EventListener for randomizer
+const getRandomShow = () => {
+  const randomIndex = Math.floor(Math.random() * shows.length);
+  return shows[randomIndex];
+};
+
```
Comment (Maria-Manuela) [inline script.js:148]: String interpolation is to the point. The code is clean and easy to read. Well done!

**Review by Maria-Manuela — COMMENTED** [review]
Project Review Overall

Strengths:
Functionality: The code appears to work properly and meets the basic requirements.
Design: The design is visually appealing and the hover effect on the cards is a nice touch.
Extra effort: I appreciate the use of your own content and ideas.
Additional features: The filter and sort options are well-implemented.

Areas for improvement:
Extra space: There seems to be unwanted extra space after the cards, which could be removed for a cleaner layout.
Footer (optional): In my opinion, adding a footer could potentially improve the overall flow and completeness of the design.

Overall, this is a good project with a solid foundation. Addressing the extra space would significantly enhance the user experience. Inspecting the elements in your browser's developer tools can help identify the source of the extra space and adjust the styles accordingly. 

Good work!

File: script.js [hunk 4]
```diff
@@ -1,406 +1,205 @@
...
+const updateShowDetails = (show) => {
+  const showCardContainer = document.getElementById('show-card-container');
+  showCardContainer.innerHTML = `
```
Comment (Maria-Manuela) [inline script.js:151]: String interpolation is to the point. The code is clean and easy to read. Well done!

File: index.html
Comment (NellaMori) [inline index.html:1]: Please format your code to have the right spaces and indentation 👍

File: script.js [hunk 5]
```diff
@@ -1,406 +1,205 @@
...
+});
+
+//Function to filter by genre
+const filterByGenre = (shows, genre) => {
+  return shows.filter(show => show.genre.toLowerCase() === genre.toLowerCase());
+};
+
+//Functions to sort by rating, year or alphabeticaly
+const sortByRating = (shows, ascending = true) => {
+  return shows.sort((a, b) => {
```
Comment (NellaMori) [inline script.js:124]: very nice!

File: style.css [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+.show-cards {
+    height: 100%;
+    padding: 16px 16px 24px 16px;
+    border-radius: 16px;
+}
+
+.show-cards:hover {
+    box-shadow: 0px 0px 30px 0px #3b3f36;
+    border: 7px solid #3b3f36;
+}
```
Comment (NellaMori) [inline style.css:101]: When you apply an effect to the card, like in this case the big border, is making all the UI around move, and this is to be avoided. A solution would be to wrap the card in a div and give the div a fixed height 👍

**Review by NellaMori — APPROVED** [review]
Hi Anton, well done finishing your Library app 📚 You followed the instructions and the app fulfils the requirements of filtering and responsiveness.

Some I improvements: there is a lot of empty space at the bottom of the page, make sure the app wraps around the element to look tidy. Also in mobile mode, the movie card should take up as much space as possible in the screen, now there's a lot of empty space. Remove the empty unused spaces in the app to improve the UI and you're good to go. 

I left some comments in the code to improve the UI, but overall you did a good job 🚀

**Pull request closed without merging**